articles/user-guide/machines/resources-estimator.md

```diff
@@ -60,6 +60,8 @@ QubitCount      2       2
 BorrowedWidth   0       0
 ```
 
+For more information about the collected metrics, see the description of the [Reported Metrics](#metrics-reported). The default configuration used for the `ResourcesEstimator` counts only `T` gates. Other configurations can be set by creating and customizing a [`QCTraceSimulator`](xref:microsoft.quantum.machines.overview.qc-trace-simulator.intro) instance.
```

Comment:
`counts only T gates` can benefit from clarifying that this statement is specific to the Depth counter.

Reply:
True, but it is actually stated a few lines above, in the table of metrics.